lib/3scale/tasks/helpers/command.rb

```diff
@@ -0,0 +1,65 @@
+module ThreeScale
+  module Tasks
+    module Helpers
+      module Command
+        module_function
```

Comment:
this shouldn't be like this for module functions?

```ruby
module_function :run
```

Reply:
Both are correct @eguzki : https://ruby-doc.org/core-2.4.1/Module.html#method-i-module_function
```
If used with no arguments, subsequently defined methods become module functions.
```